conda/recipes/versions.yaml

```diff
@@ -89,7 +89,7 @@ jupyterlab_version:
 jupyter_packaging_version:
   - '>=0.7.0,<0.8'
 librdkafka_version:
-  - '=1.5.*'
+  - '=1.6.*'
```

Comment:
LFTM. We only use core librdkafka features that have not changed in several releases so a bump like this is no issue.